infra/ci_fuzz.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 Google LLC
```

Comment:
In another CL:
Could you please move all of the code you have worked on into subdirectories of infra/.
I don't think it's good to have files with names like "helper" there.

Reply:
Noted.